portfolio/src/main/webapp/script.js

```diff
@@ -16,28 +16,70 @@ const VISIBLE = 'visible';
 const HIDDEN = 'hidden';
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random polaroid image to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 
-      'You know what\'s funnier than 24? 25!', 
-      'East?! I thought you said Weast!', 
-      'Squidward, I used your clarinet to unclog my toilet!'];
-
-  // Picks a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Adds the random greeting to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function addRandomPolaroid() {
+  const images =
+      ['arches.JPG','cruise.jpg','dalat.jpg', 'mexico.jpg', 
+      'pittsburghSunset.jpg', 'saigonLightening.jpg', 
+      'saigonTower.jpg', 'thailand.jpg', 'yellowstone.JPG'];
+
+  // Picks a random image.
+  const image = images[Math.floor(Math.random() * images.length)];
+  
+  // Creates a div tag to store the polaroid image and description.
+  let div = document.createElement("div");
+  div.setAttribute('class', 'polaroid');
+
+  // Creates a p tag that contains the image name.
+  const imageName = image.split(".")[0];
+  let imageText = createPTag(imageName);
+
+  // Creates an img tag to store the image.
+  let imgTag = createImgTag(image);
+
+  div.appendChild(imgTag);
+  div.appendChild(imageText);
+  
+  // Adds the polaroid div tag to the page.
+  const imagesContainer = document.getElementById('images');
+  imagesContainer.appendChild(div);
+}
+
+/**
+ * Creates an img tag to store the image.
+ * @param {string} image The name of the image
+ *     file to create an img tag for.
+ * @return {object} This returns an img tag using
+ *     the given image.
+ */
+function createImgTag(image) {
+  let imgTag = document.createElement("img");
+  imgTag.setAttribute("src", 'images/' + image);
```

Comment:
the path "images" seems to be used here and on line 45. It would be good to store this in a global const because if you ever change the directory you do not need to change it in multiple places.

Reply:
The 'images' in this line refer to a folder while the images in line 45 refer to a HTML element. Since they refer to different things, should I still create a global const for them?

Reply:
(I updated this is in pull request #19)

Done!